src/store/users/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface IUser {
+  id: number;
+  name: string;
+  userName: string;
+  email: string;
+  address: string;
+};
+
+export type IDataApi = {
```

Comment:
I would suggest adding other fields of data API. if I'm not mistaken, there should be config and smth else